### PR TITLE
Add tests/integration to the presubmit

### PR DESCRIPTION
**Pull request by aiuto**
Adds the hello_bazel integration tests to the presubmit.

### CODE REVIEW

**Comment by aiuto** [comment]
Is there documentation about how CI works somewhere?
There is a log of magic in the .yml files.
The big thing that baffles me is how I run the CI stream from my git client, without having to do a presubmit. That should be a basic feature.

File: .bazelci/tests.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+tasks:
+  hello_macos:
+    name: hello
+    working_directory: integration/hello_bazel
+    platform: macos
+    # I realy want bazel run. Is that allowed?
```
Comment (fweikert) [inline .bazelci/tests.yml]: use run_targets for bazel run (https://github.com/bazelbuild/continuous-integration/tree/master/buildkite#running-commands-shell-scripts-or-binary-targets)

File: .bazelci/tests.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    # Can we run arbitrary commands? It is limiting to only allow bazel
```
Comment (fweikert) [inline .bazelci/tests.yml]: Yes, shell_commands (linux + macos) or batch_commands (windows): https://github.com/bazelbuild/continuous-integration/tree/master/buildkite#running-commands-shell-scripts-or-binary-targets

**Review by fweikert — COMMENTED** [review]
Documentation: https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/README.md#basic-syntax

The pipeline works based on the GitHub repo, and to my knowledge there is no good way of running tests locally (There is https://buildkite.com/changelog/44-run-pipelines-locally-with-bk-cli, but it's for a different use case).

File: .bazelci/tests.yml (hunk 1 above)
Comment (aiuto) [reply .bazelci/tests.yml]: Done.

File: .bazelci/tests.yml (hunk 2 above)
Comment (aiuto) [reply .bazelci/tests.yml]: ack.

**Pull request merged**